app/components/simple-editor.js

```diff
@@ -92,6 +93,75 @@ import {
   text_variable,
   textVariableView,
 } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/variable-plugin/variables';
+
+/**
+ * If the user config is present, merge it with the default config.
+ * Otherwise return the default config.
+ * @param defaultConfig
+ * @param userConfig
+ * @returns {*}
+ */
+const maybeMergeUserConfig = (defaultConfig, userConfig) => {
```

Comment:
Honestly, I think just calling this "mergeConfigs" is fine. There is no maybe: if the userConfig is empty, it also gets "merged", there is just nothing to merge :shrug: